fili-core/src/main/java/com/yahoo/bard/webservice/web/FiliResponseWriter.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+
+/**
+ * Default writer for response. Will choose to use the writer type from format type.
+ */
+public class FiliResponseWriter implements ResponseWriter {
+
+    private final FiliResponseWriterSelector responseWriterSelector = new FiliResponseWriterSelector();
+
+    /**
+     * Selects a writer from mapping and writes response.
+     *
+     * @param request  ApiRequest object with all the associated info in it
+     * @param responseData  Data object containing all the result information
+     * @param os  OutputStream
+     *
+     * @throws IOException if a problem is encountered writing to the OutputStreamC
+     */
+    public void write(
```

Comment:
Also, since this method isn't changing the contract of the method it's implementing, JavaDoc isn't necessary.